.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "rocketmq-apis"]
```

Comment:
Although we have placed IDL in [rocketmq-apis](https://github.com/apache/rocketmq-apis), I still don't think that rocketmq-apis is an appropriate name to describe IDL. "apis" seems more like a description for the interfaces of concrete code implementation. Therefore, it might be more fitting to place the submodule in a directory called "proto" or "IDL"?

Reply:
Agree with your insight. I renamed the submodule to `protos`.